src/user-guide/installing-workflows.rst

```diff
@@ -392,8 +392,22 @@ If:
 
   .. autoclass:: cylc.flow.unicode_rules.WorkflowNameValidator
 
```

Comment:
I wonder if this whole section should be re-worded slightly so as to frame it in terms of disallowed things, as opposed to "expected errors". It can wait though.